spec/views/collections/_form_for_select_collection.html.erb_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe 'collections/_form_for_select_collection.html.erb' do
+  let(:collections) {
+    [
+      Collection.new(id: '1234', title: 'collection 1', create_date: DateTime.parse('Thu, 13 Aug 2015 14:20:22 +0100')),
+      Collection.new(id: '1235', title: 'collection 2', create_date: DateTime.parse('Thu, 13 Aug 2015 14:18:22 +0100')),
+      Collection.new(id: '1236', title: 'collection 3', create_date: DateTime.parse('Thu, 13 Aug 2015 14:16:22 +0100')),
+      Collection.new(id: '1237', title: 'collection 4', create_date: DateTime.parse('Thu, 13 Aug 2015 14:29:22 +0100'))
+    ]
+  }
+  let(:solr_collections) {
+    collections.map do |c|
+      SolrDocument.new(c.to_solr).tap do |sd|
+        sd['system_create_dtsi'] = c.create_date.to_s
+      end
+    end
+  }
+
+  let(:doc) { Nokogiri::HTML(rendered) }
+
+  before do
+    allow(view).to receive(:user_collections).and_return(solr_collections)
+  end
+
+  it "sorts the collections" do
+    render
```

Comment:
try assigning render to a `let!` variable so it might only render the page once.